canvas-plugins/metriport-app/README.md

```diff
@@ -0,0 +1,22 @@
+# metriport-app
+
+## Description
+
+Get instant access to comprehensive patient clinical history summaries, and more.
+
+### Installation
+
+Requirments:
+
+1. [Canvas CLI](https://docs.canvasmedical.com/sdk/canvas_cli/) set up
+2. Metriport Token
+
+Please reach out to support@metriport.com to get a Metriport Token if not already provided.
+
+1. Install the plugin: `canvas install metriport-app`
```

Comment:
Add a "Steps" title here? The "Please reach out..." is related to the requirements section. Then we start a new section w/o a new section title.